README.md

```diff
@@ -96,10 +96,18 @@ Alternatively, you can use **extensions** (such as prettier or black) to format
 
 There are three linters/formatters for Python: `flake8`, `isort`, `black`; and there are two for Javascript/TypeScript: `prettier`, `eslint`.
 
-In order to install pre-commit git-hook, run commands (recommended with [venv](https://docs.python.org/3/library/venv.html) previously created):
+In order to install pre-commit git-hook, run command
+```
+scripts/install.sh
+```
```

Comment:
After a clean clone, this doesn't work. Running `chmod +x scripts/install.sh` fixes this and should either be specified in this readme or the permissions of the file should be committed as well.

Reply:
I commited permissions for the script

Reply:
Looks okay now